includes/login-register.php

```diff
@@ -31,6 +31,12 @@ function edd_login_form( $redirect = '' ) {
 
 	ob_start();
 
+	if ( ! empty( edd_get_errors() ) ) {
```

Comment:
PHPunit error (Travis) for this line: `PHP Fatal error:  Can't use function return value in write context in /home/travis/build/easydigitaldownloads/easy-digital-downloads/includes/login-register.php on line 34`